core/filter_constants_inc.php

```diff
@@ -67,3 +77,13 @@
 define( 'FILTER_PROPERTY_PLATFORM', 'platform' );
 define( 'FILTER_PROPERTY_OS', 'os' );
 define( 'FILTER_PROPERTY_OS_BUILD', 'os_build' );
+
+# These are old date fields names, keeping them for backward compatibility, if used externally
+define( 'FILTER_PROPERTY_START_DAY', FILTER_PROPERTY_DATE_SUBMITTED_START_DAY );
```

Comment:
Right, as atrol says, having now both filter properties for "date submitted" and "last updated", would make the original constant names ambiguous. Previous constants are still kept for now for compatibility.

> wouldn't it be easier to understand if the new constants contain also "DATE", e.g. FILTER_PROPERTY_DATE_LAST_UPDATED_START_DAY

Actually i used the field/column names, with are exactly "date submitted", "last updated", as most of the properties follow the technical names.
If you agree that it's clear by adding DATE explicitly, the change is trivial though.